autoRSA.py

```diff
@@ -31,6 +31,8 @@
     from tradierAPI import *
     from vanguardAPI import *
     from webullAPI import *
+    from sofiAPI import *
```

Comment:
These references to sofiAPI need removed. I see this on line 34 as well as line 59, and line 71.

Reply:
Oops, I thought I removed them